app/database_etl/location_utils/location_util_functions.py

```diff
@@ -76,4 +81,66 @@ def get_city(row):
             return [f"{city},{row['state'][0]}" for city in cities]
         return cities
     else:
-        return row['city']
\ No newline at end of file
+        return row['city']
+
+# Checks whether each row of an input city or state dataframe
+# is in one of the WHO's disputed areas
+def check_if_in_disputed_area(df: pd.DataFrame) -> pd.DataFrame:
+    WHO_FL_URL = "https://services.arcgis.com/5T5nSi527N4F7luB/arcgis/rest/services/DISPUTED_AREAS_mask/FeatureServer/0"
+    # Create feature layer object
+    disputed_areas_fl = FeatureLayer(WHO_FL_URL)
+
+    # Defining this as a closure so that we have access to
+    # disputed_areas_fl
+    def row_in_disputed_area(row: pd.Series) -> bool:
```

Comment:
I know why you defined it as a closure but I'm not rly a huge fan of this pattern tbh. You could just pass in disputed areas fl like you did below when you passed in the geo_dfs